src/app/components/app.component.ts

```diff
@@ -36,10 +36,10 @@ import {Home} from './home/home';
   `
 })
 @RouteConfig([
-  { path: '/', component: Home, name: 'Index' },
-  { path: '/home', component: Home, name: 'Home' }
+  { path: '/', component: HomeCmp, name: 'Index' },
+  { path: '/home', component: HomeCmp, name: 'Home' }
 ])
-export class App {
+export class AppCmp {
```

Comment:
Certainly is the going standard out there.
I guess this PR is almost more for discussion :)

I just shutter at the thought of naming all components with full suffix `Component`.... just for simplicity sake. When shorthand is ample, why not I guess.